package.json

```diff
@@ -25,6 +26,11 @@
     "eslint-plugin-unicorn": "^57.0.0",
     "globals": "^16.0.0",
     "prettier": "^3.5.3",
-    "vite": "^4.4.5"
+    "vite": "^4.4.5",
+    "husky": "^8.0.0"
+  },
+  "lint-staged": {
+    "**/*.{js,jsx,ts,tsx}": "eslint --fix",
+    "**/*.{css,scss,md,json}": "prettier --write"
   }
```

Comment:
It’s better not to use the `--fix` option when running `lint-staged` because its purpose is to check for issues in the code before committing, not to automatically fix them. If `lint-staged` detects any problems, the developer should review and fix them manually. This ensures they are aware of the issues and can make intentional changes rather than relying on automatic fixes that might not always produce the expected results.